plugins/module_utils/_ADObject.psm1

```diff
@@ -611,10 +614,17 @@ Function ConvertTo-AnsibleADDistinguishedName {
                 continue
             }
 
-            $adDN = Get-AnsibleADObject @getParams |
-                Select-Object -ExpandProperty DistinguishedName
-            if ($adDN) {
-                $results.Add($adDN)
+            $object = Get-AnsibleADObject @getParams
+            if ($object) {
+                if ($NestedGroupFlatten -and $object.ObjectClass -eq "group") {
+                    $dns = Get-ADGroupMember $object -Recursive | Select-Object -ExpandProperty DistinguishedName
```

Comment:
We want to also splat `getParams` here so it will run with the same `-Server` and `-Credential` options set in the module.

Reply:
Done